.gitignore

```
@@ -8,3 +8,4 @@ services/*/api/
 pkg
 .certs
 .idea/
+spec/services/*/oas3.patched
```

For confirmation, this is the proposed change?

```suggestion
spec/services/eiav2/oas3.patched
spec/services/fabricv4/oas3.patched
```

(and then the removal of only those patch directories, leaving the metalv1/oas3.patched directory in repo)